src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -30876,13 +30886,19 @@ static void F(C<object?> x, C<object> y)
                 // (8,13): warning CS8619: Nullability of reference types in value of type 'C<object?>' doesn't match target type 'I<object>'.
                 //         a = x;
                 Diagnostic(ErrorCode.WRN_NullabilityMismatchInAssignment, "x").WithArguments("C<object?>", "I<object>").WithLocation(8, 13),
+                // (9,13): warning CS8619: Nullability of reference types in value of type 'C<object?>' doesn't match target type 'I<object>'.
+                //         a = x!;
+                Diagnostic(ErrorCode.WRN_NullabilityMismatchInAssignment, "x!").WithArguments("C<object?>", "I<object>").WithLocation(9, 13),
```

Comment:
More general curiosity: how would one silence this warning, if not by the `!`?

Reply:
> More general curiosity: how would one silence this warning, if not by the !?

I'll open an issue for this question

---
In reply to: [218172518](https://github.com/dotnet/roslyn/pull/29947#discussion_r218172518) [](ancestors = 218172518)

Reply:
I think [https://github.com/dotnet/roslyn/issues/29642](https://github.com/dotnet/roslyn/issues/29642) tracks this already

---
In reply to: [218174205](https://github.com/dotnet/roslyn/pull/29947#discussion_r218174205) [](ancestors = 218174205,218172518)

Reply:
> I think [https://github.com/dotnet/roslyn/issues/29642](https://github.com/dotnet/roslyn/issues/29642) tracks this already

Agree 

---
In reply to: [218209547](https://github.com/dotnet/roslyn/pull/29947#discussion_r218209547) [](ancestors = 218209547,218174205,218172518)